lib/9pfs/9pfs_vnops.c

```diff
@@ -601,5 +602,6 @@ struct vnops uk_9pfs_vnops = {
 	.vop_cache	= uk_9pfs_cache,
 	.vop_fallocate	= uk_9pfs_fallocate,
 	.vop_readlink	= uk_9pfs_readlink,
-	.vop_symlink	= uk_9pfs_symlink
+	.vop_symlink	= uk_9pfs_symlink,
+	.vop_poll	= uk_9pfs_poll
```

Comment:
Maybe add a comma at the end to make this compatible with the other `poll` fields.